src/app/styles.jsx

```diff
@@ -4,3 +4,4 @@ export const Wrapper = styled.section`
   background-color: ${props => props.theme.colors.background};
   /* padding: 4em; */
 `;
+export default Wrapper;
```

Comment:
It is not needed as it's already been exported before.

Reply:
Done